pkgs/top-level/perl-packages.nix

```
@@ -23306,6 +23340,19 @@ let
     };
   };
 
+  TestWeaken = perlPackages.buildPerlPackage {
+    pname = "Test-Weaken";
+    version = "3.022000";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/K/KR/KRYDE/Test-Weaken-3.022000.tar.gz";
+      sha256 = "2631a87121310262e0e96107a6fa0ed69487b7701520773bee5fa9accc295f5b";
+    };
```

```suggestion
    };
    propagatedBuildInputs = [ ScalarListUtils ];
```

per https://metacpan.org/release/KRYDE/Test-Weaken-3.022000/source/Makefile.PL#L21